sonic-net/sonic-buildimage
# [Netberg][Broadcom] Added support for Aurora 221

#### Why I did it

Support for  Broadcom Trident 3 based platforms Netberg Aurora 221
ASIC: Trident 3 Broadcom BCM56275
Pors: 48x 1G BASE-T, 8x 10G SFP+ ,


##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Added specification to device/netberg directory with pddf description 
Added platform/broadcom/sonic-platform-modules-netberg contains scripts and sonic_platform packages.
Added platform/broadcom/platform-modules-netberg.mk to include Aurora 221 related IDs
Modified rules.dep, rules.mk, one-image.mk in platform/broadcom/rules.dev

#### How to verify it

Build SONiC

Install the image on the device and verify the related components are installed and shown correctly.

Verification output - [221-show.txt](https://github.com/sonic-net/sonic-buildimage/files/15087609/221-show.txt)



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)



- [x] master (19e3bbfb11b70a2d145e135b05df065bc765fe6f)

#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
